cni/pkg/install/kubeconfig.go

```diff
@@ -120,6 +120,11 @@ func createKubeconfigFile(cfg *config.InstallConfig, saToken string) (kubeconfig
 		return "", err
 	}
 
+	// When using Multus, the net.d dir might not exist yet, so we must create it
+	if err := os.MkdirAll(cfg.MountedCNINetDir, os.FileMode(0o755)); err != nil {
```

Comment:
shouldn't this only return an error `if !os.IsExist(err)`?

Reply:
From the docs: 

> If path is already a directory, MkdirAll does nothing and returns nil.

Reply:
:facepalm: thanks